src/component/Invoice/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react"
+import moment from "moment"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons"
+
+function Invoice({ invoiceTotal, invoiceStatus, invoiceStatusName, clientName, invoiceId, dueDate }) {
+
+    const date = moment(dueDate);
+    const formattedDate = date.format("DD MMM YYYY");
+    const element = <FontAwesomeIcon icon={faChevronRight} />
+
+
+    useEffect(setColor, [])
+
+    const [statusColor, setStatusColor] = useState('')
+    const [statusBorderColor, setStatusBorderColor] = useState('')
+
+    function setColor() {
+        if (invoiceStatus === "Paid") {
+            setStatusColor("text-green-500")
+            setStatusBorderColor("border-green-500")
+        } else if (invoiceStatus === "Pending") {
+            setStatusColor("text-yellow-400")
+            setStatusBorderColor("border-yellow-400")
+        } else if (invoiceStatus === "Overdue") {
+            setStatusColor("text-red-600")
+            setStatusBorderColor("border-red-600")
+        } else {
+            setStatusColor("text-zinc-500")
+            setStatusBorderColor("border-zinc-500")
+        }
```

Comment:
You could simplify this function and reduce the amount of state you're having to deal with.

With each status, you're having to update 2 bits of state.

What if your state was just the colour ie red-600? You could then add text- or border- in your JSX